test/Microsoft.NET.Build.Tests/GivenThatWeWantToReferenceAnAssembly.cs

```diff
@@ -26,6 +26,13 @@ public class GivenThatWeWantToReferenceAnAssembly : SdkTest
             string referencerTarget,
             string dependencyTarget)
         {
+            if (UsingFullFrameworkMSBuild)
```

Comment:
I don't understand why we are making this change here.  It seems irrelevant to your current feature work and shouldn't be modified.